src/wire/TerribleCommandBuffer.cpp

```diff
@@ -43,7 +45,7 @@ namespace nxt { namespace wire {
     }
 
     void TerribleCommandBuffer::Flush() {
-        mHandler->HandleCommands(mBuffer, mOffset);
+        NXT_ASSERT(mHandler->HandleCommands(mBuffer, mOffset));
```

Comment:
Do you think it should just return `bool` or should we use the Result.h/Error.h stuff out here?

Reply:
I think we should just use bool instead of Result<void, void>. It is more self explanatory (though it doesn't explicitly show which of true or false is the success value).